ios/RNStarPrnt.m

```diff
@@ -534,8 +534,15 @@ -(void)appendCommands:(ISCBBuilder *)builder
             BOOL diffusion = ([[command valueForKey:@"diffusion"] boolValue] == NO) ? NO : YES;
             BOOL bothScale = ([[command valueForKey:@"bothScale"] boolValue]  == NO) ? NO : YES;
             SCBBitmapConverterRotation rotation = [self getBitmapConverterRotation:[command valueForKey:@"rotation"]];
+            NSError *error = nil;
             NSURL *imageURL = [NSURL URLWithString:urlString];
-            NSData *imageData = [NSData dataWithContentsOfURL:imageURL];
+            NSData *imageData = [NSData dataWithContentsOfURL:imageURL options:NSDataReadingUncached error:&error];
+
+            if (error != nil) {
```

Comment:
Is this converting a general file path to a `file:` URL by detecting a parse error when interpreting it as a general URL? If so, should more specific `NSError` type be detected or at least a comment be added to explain it? Could it accidentally catch unrelated errors?